pyrightconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "include": [
+    "main.py"
+  ],
+
+  "exclude": [
+    "**/node_modules",
```

Comment:
Ok I am now biased against this for the sole reason that it uses node.js.
This is a simple one small python file discord bot. Don't make people install a huge node modules folder to develop it.

Do your type annotations work with other linters?

Reply:
Sorry, I took a lot of the config from a sample config file and didn't understand what this meant. This is my fault and I can remove this.